---
# The current state of all controller
rover_main/controllerMap controllerState